### ISSUE TITLE
betanet RPC

### ISSUE THREAD

**Issue by Mentors4EDU** [BitBadges/bitbadgeschain#2]
Live testnet RPC

**Comment by Mentors4EDU** [comment]
This will be interesting, the CosmWasm implementation might help.

**Comment by trevormil** [comment]
CosmWASM should be done now (https://docs.bitbadges.io/for-developers/tutorials/create-a-smart-contract).

GitBook docs are also super updated now in all categories. Would recommend a read through so you can learn about tech stack and we are on same page for everything.

Realistically, we are ready for a betanet launch when we feel like it (tech / development wise). We have a solid MVP. Will just need ~1 week on my end to get everything ready with regards to blockchain setup and learn about production deployment. 

However, I do still feel like there is a good amount of stuff still needed to be done before launch (socials, marketing, legal stuff, additional features, etc). I think betanet launch should be launched with not just a rough MVP but a really complete and almost final product. I think taking it slower rather than faster is a better choice.

**Comment by Mentors4EDU** [comment]
I agree I just make the GitHub issues for what is pending project wise. Production is not ready until socials + other stuff are figured out.
